README.md

```diff
@@ -265,6 +236,87 @@ The static site that gets built is pushed to the `documentation` branch of this
 We host the site on netlify. On master branch builds (see `.github/workflows/documentation.yml`), we push the built docs to
 the `documentation` branch. Netlify automatically re-deploys the docs pages whenever there is a change to that branch.
 
+## Releases
+### Release Timeline for minor releases
+**For Rasa Open Source, we usually commit to time-based releases, specifically on a monthly basis.** 
+This means that we commit beforehand to releasing a specific version of Rasa Open Source on a specific day, 
```

Comment:
?
```suggestion
This means that we commit beforehand to release a specific version of Rasa Open Source on a specific day, 
```

Reply:
I believe the correct way is `releasing` 😅  maybe we need a native speaker to break the tie here ahah

Reply:
Consider this as non-blocking 😆 
@erohmensing Help please 😄

Reply:
honestly i think both could work, one as a noun and one as a verb -- "we commit to [the act of] releasing a specific version" or "we commit to release a specific version" 

that being said i'd probably go with the noun. "this year i'm committing to running three times a week" 

but now i've thought about it too much and the word looks fake 😛

Reply:
Ok so I'm leaving it as is 😂